### DISCUSSION TITLE
Steps to debug the event flows

### DISCUSSION THREAD

**Discussion by TestCK** [enisdenjo/graphql-sse#19] [Q&A] [answered]
Is there any way to debug the events coming from server to client.?

I tried looking at the network tab of the browser(chrome), don't seem to get any logs there.

It will be helpful if there is a way to see all the events from server for debugging

**Comment by enisdenjo** [comment, marked as answer]
Moved to https://github.com/enisdenjo/graphql-sse/issues/20.
